docker-compose.arm64.yml

```diff
@@ -0,0 +1,129 @@
+version: '3'
+
+# Profiles by name:
+#   - all (webpack, python, worker, redis, postgres)
+#   - ee_all (webpack, python, worker, plugins, redis, postgres, clickhouse, kafka, zookeeper)
+#   - or, list profiles desired:
+#            docker compose -f docker-compose.arm64.ch.yml \
+#                up
+#                python \
+#                redis \
+#                postgres
+
+services:
+    postgres:
+        image: postgres:12-alpine
+        container_name: posthog_db
+        environment:
+            POSTGRES_USER: posthog
+            POSTGRES_DB: posthog
+            POSTGRES_PASSWORD: posthog
+        ports:
+            - '5432:5432'
+    redis:
+        image: 'redis:5-alpine'
+        container_name: posthog_redis
+        ports:
+            - '6379:6379'
+    python:
+        build:
+            context: .
+            dockerfile: dev.Dockerfile
+        command: ./bin/docker-backend
+        volumes:
+            - .:/code
+        environment:
+            DATABASE_URL: 'postgres://posthog:posthog@postgres:5432/posthog'
+            REDIS_URL: 'redis://redis:6379/'
+            SECRET_KEY: '<randomly generated secret key>'
+            DEBUG: 'true'
+            PGHOST: postgres
+            PGUSER: posthog
+            PGPASSWORD: posthog
+        depends_on:
+            - postgres
+            - redis
+        links:
+            - postgres:postgres
+            - redis:redis
+        ports:
+            - '8000:8000'
+    clickhouse:
+        build:
+            context: .
+            dockerfile: clickhouse.arm64.Dockerfile
+        command: clickhouse-server -C /etc/clickhouse-server/config.xml
+        volumes:
+            - /var/lib/clickhouse/
+        ports:
+            - '8123:8123'
+            - '9000:9000'
+            - '9440:9440'
+            - '9009:9009'
+            - '8000:8000'
```

Comment:
you have this defined twice which will break the compose config

Reply:
yeah, that was for networking testing, it's gone now